### DISCUSSION TITLE
run on flatpak cant follow the instructions in the wiki find rom in app

### DISCUSSION THREAD

**Discussion by jcleng** [mamedev/discussions#41] [Q&A]
```
[📦 org.mamedev.MAME ~]$ mame -version
0.254 (unknown)mame: ../../src/fccache.c:808: FcCacheFini: Assertion `fcCacheChains[i] == NULL' failed.
已放弃（核心已转储）
```

https://flathub.org/zh-Hans/apps/org.mamedev.MAME
```
vi ~/.mame/mame.ini
.....
rompath                   /home/jcleng/下载/mame-mslug/
....
```
```
ls /home/jcleng/下载/mame-mslug/
mame-mslug.zip
```

[![p9cvuR0.png](https://s1.ax1x.com/2023/05/14/p9cvuR0.png)](https://imgse.com/i/p9cvuR0)

**Comment by startaq** [comment]
MAME does not officially support Flatpak.

You can ask your question here: https://github.com/flathub/org.mamedev.MAME

**Comment by jflatt** [comment]
The ROM should be named mslug.zip.  You also need to have neogeo.zip BIOS files in there too.

**Reply by jcleng** [reply]
nice, its working, i get neogeo.zip file of bios,and search name with playlist name in app to download

```
tree /home/jcleng/下载/mame-mslug/
/home/jcleng/下载/mame-mslug/
├── dinou.zip
├── dino.zip
├── kov3.zip
├── kov.zip
├── mslug.zip
└── neogeo.zip

0 directories, 6 files

```
